app/view_objects/comments/tree.rb

```diff
@@ -0,0 +1,28 @@
+module Comments
```

Comment:
Love the introduction of view objects 💃

Reply:
Thanks! Though I'm not sure if I'll keep it. I'm concerned about the memory usage cause of instantiating more objects (I'm checking it now)

Reply:
:-) in general there's a lot of logic in the views. It doesn't hurt to slowly make them more declarative anyhow

Reply:
Is this similar to the decorators we have in `/decorators` folder?

Reply:
@maestromac they are similar in some terms. But they are also different: decorators are usually used to add functionality to an object, the draper ones that exist on a project add (often view-related) functionality to an instance of a model class. View objects are not tied to a specific object and contain view logic. They are often called presenters, but in my opinion this name is a bit confusing. I like the explanation of a view object from [a (classic) CodeClimate article](https://codeclimate.com/blog/7-ways-to-decompose-fat-activerecord-models/)

Reply:
Probably I'll have to remove the view object from this pr, but I think this concept could be used for refactoring in other places.

Reply:
Thanks for the explanation. We definitely benefit from having some view objects